roles/bareos-database/tasks/postgresql_install.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+# APT module does not updating the cahce by default and since
+# package module used instead, the cache should be updated previously.
+- name: Update apt cache
+  apt:
+    update_cache: yes
+  when: ansible_os_family == 'Debian'
+
+- name: Install Postgres
+  package:
+    name: '{{ item }}'
+    state: present
+  with_items: '{{ postgres_packages }}'
+  register: bareos_install
+
+- name: Init new Postgresql db cluster on RHEL/CentOS 7
+  command: /usr/bin/postgresql-setup initdb
```

Comment:
Can we use a module instead?

Reply:
Unfortunately, "initdb" command is not supported by any ansible module.